src/implementing_new_features.md

```diff
@@ -79,7 +78,6 @@ feature can be changed, or the feature might be completely
 rewritten or removed. Features are not supposed to gain tenure
 by being unstable and unchanged for a year.
 
-<a name = "tracking-issue"></a>
```

Comment:
Removing this anchor will technically break links, but it seems unlikely that people are linking to this specific section that much.